Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -525,17 +526,21 @@ def write(self, writer, font, tableDict, value, repeatIndex=None):
 
 class Table(Struct):
 
-	longOffset = False
 	staticSize = 2
 
 	def readOffset(self, reader):
 		return reader.readUShort()
```

Comment:
Suggest either making readOffset support all sizes - and not implementing it on subclasses - or making the subclasses implement writeNullOffset.

I prefer the subclass implements and the parent should assert staticSize == 2 in both readOffset and writeNullOffset